book/website/community-handbook/working-groups.md

```diff
@@ -0,0 +1,34 @@
+(ch-working-groups)=
+# *The Turing Way* Working Groups
+
+Working Groups (WGs) are used across many sectors including government, academia, business and nonprofit or volunteer-led organisations to decentralise operations and decision making.
```

Comment:
This is a personal thing, but I don't like the use of Working Groups = WGs. It's incredibly easy for people, particularly newer community members, to get bogged down in acronyms and abbreviations and I think we should be mindful of that in documentation that governs our human infrastructure and processes, since we want to make them accessible to people.